content_script_all_frames.js

```diff
@@ -60,15 +60,20 @@ function setFavIconBadge(settings) {
 }
 
 function runFunction(f, context) {
-    var doc;
+    let doc = document?.querySelector('#gsft_main')?.contentWindow?.document;
```

Comment:
That's some modern syntax, cool ;)

Reply:
Well, apparently it doesn't actually work if `document` is undefined, so this might still fail if `document` for some reason isn't available 😞